commodity_trading/data/res_config_settings.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="res_config_settings_multi_step_routes" model="res.config.settings">
+        <field name="group_product_variant" eval="0"/>
```

Comment:
bydefault it is disabled?

Reply:
By default it is enabled and as per the db it is not enabled so we have made it disabled